coalib/settings/ConfigurationGathering.py

```diff
@@ -47,7 +47,7 @@ def load_config_file(filename, log_printer, silent=False):
     :param log_printer: The log printer to log the warning/error to (in case).
     :param silent:      Whether or not to warn the user/exit if the file
```

Comment:
in a nutshell, if youre lazy, try: ConfigurationGather: Fix typo in docstring

Fixes a typo in a docstring of the ``load_config_file function``.

Fixes url